libraries/botbuilder-core/src/cardFactory.ts

```diff
@@ -276,6 +285,15 @@ export class CardFactory {
         buttons?: (CardAction | string)[],
         other?: Partial<ThumbnailCard>
     ): Attachment;
+    /**
+     * Returns an attachment for a thumbnail card.
+     *
```

Comment:
same here

Reply:
It's like this in the previous overloads of this method, that's why I kept it.